devito/interfaces.py

```diff
@@ -159,6 +159,12 @@ def _indices(cls, **kwargs):
                 dimensions = [symbols("x%d" % i) for i in range(1, len(shape) + 1)]
         return dimensions
 
+    def _cached_init(self):
```

Comment:
Ok, so as mentioned in the chat, this fix will most definitely cause memory bleeds due to SymPy's caching causing us the invalidate the "original" symbol instance as soon as we form a derivative. This means we will never deallocate any memory, so this needs to be pulled.

However, I would argue that fixing the caching bug should not be part of this PR and instead we can simply `xfail` the new TTI test, which will hopefully trigger on this in travis.